gemrb/GUIScripts/pst/GUIINV.py

```diff
@@ -385,14 +385,20 @@ def OnAutoEquip ():
 	UpdateInventoryWindow ()
 	return
 
-def OnDragItem ():
-	pc = GemRB.GameGetSelectedPCSingle ()
+def OnDragItem (btn, slot):
+	"""Updates dragging."""
 
-	slot, slot_item = ItemHash[GemRB.GetVar ('ItemButton')]
+	#don't call when splitting items
+	if ItemAmountWindow != None:
+		return
+
+	pc = GemRB.GameGetSelectedPCSingle ()
+	slot_item = GemRB.GetSlotItem (pc, slot)
 
 	if not GemRB.IsDraggingItem ():
-		ResRef = slot_item['ItemResRef']
-		item = GemRB.GetItem (ResRef)
+		if not slot_item:
```

Comment:
When does this happen?

Reply:
IIRC, I placed this here to silence a python error when you try and 'drag' from an empty slot, but its possible that the root cause that no longer exists, so I will test again without it.

Reply:
Deserves a comment then.

Reply:
It appears UpdateSlot is careful enough to avoid this, I removed the check